### DISCUSSION TITLE
Did Uint8List become unsupported with 3.0.0-dev.9?

### DISCUSSION THREAD

**Discussion by hedbladucf** [isar/isar#578] [Help]
Hi,

I was wondering if the "Uint8List" type became unsupported with the latest dev-release? Once I upgraded, build-runner is now saying that I have to add the "@ignore" annotation to the field in order for build_runner to successfully run. Maybe I missed where you mentioned this, or is Uint8List no longer supported?

**Comment by simc** [comment]
Yes, sorry I forgot to mention this in the changelog. Please use `List<byte>` instead of `Uint8List` as property type. You can still assign `Uint8List`s.

**Reply by hedbladucf** [reply]
Ah, got it - thank you, Simon! Btw, I love your projects :)
